llvm-spirv/include/LLVMSPIRVExtensions.inc

```diff
@@ -49,3 +49,4 @@ EXT(SPV_INTEL_bfloat16_conversion)
 EXT(SPV_INTEL_joint_matrix)
 EXT(SPV_INTEL_hw_thread_queries)
 EXT(SPV_INTEL_global_variable_decorations)
+EXT(SPV_KHR_uniform_group_instructions)
```

Comment:
I don't sure if we need it here, until the extension is ready and transfered to this repository: https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1253. The same about Clang.cpp. @MrSidims WDYT?

Reply:
@aobolensk please remove this change from the translator, let the translator's work be merged and pulled to intel/llvm first.